app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,5 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="blue_hacktzi">#2A3B5A</color>
```

Comment:
Good naming for resources, but I think we can talk with designer in order to get the palette of colors and it's names :). It is a good practice, because the team can manage just one single name for resources, event if the member team is designer, developer, scrum master, etc :)